src/main/java/org/sunbird/cb/hubservices/serviceimpl/ProfileService.java

```diff
@@ -82,4 +90,27 @@ public Response multiSearchProfiles(String userId, MultiSearch mSearchRequest, S
 		return response;
 	}
 
+	@Override
+	public SBApiResponse getRelationshipBetweenUsers(String toUserId, String authToken) {
+		SBApiResponse response = ProjectUtil.createDefaultResponse(Constants.API_USER_RELATIONSHIP);
+		String fromUserId = "";
+		try {
+			fromUserId = accessTokenValidator.fetchUserIdFromAccessToken(authToken, response);
+			if (StringUtils.isEmpty(fromUserId)) {
+				return response;
+			}
+			Map<String, String> map = connectionService.getRelationshipBetweenUsers(fromUserId,
+					toUserId);
+			response.getResult().put("response",map);
+			response.getParams().setStatus(Constants.OK);
+			response.setResponseCode(HttpStatus.OK);
```

Comment:
this is already done in createDefaultResponse method -- let's keep that value...